File: 1066.py
a = int(input())
b = int(input())
c = int(input())
d = int(input())
e = int(input())

par=0
imp=0
po=0
ne=0

for n in [a,b,c,d,e]:
  if n % 2 ==0:
    par = par + 1
  else:
    imp = imp + 1
  if n > 0:
    po = po + 1
  if n < 0:
    ne = ne+1

print('{} valor(es) par(es)'.format(par))
print('{} valor(es) impar(es)'.format(imp))
print('{} valor(es) positivo(s)'.format(po))
print('{} valor(es) negativo(s)'.format(ne))

File: CSharp/1066.cs
using System; 

class URI {

    static void Main(string[] args) 
    {
        int par=0;
        int impar=0;
        int positivo=0;
        int negativo=0;
        int num;

        for(int i=0;i<5;i++)
        {
            num = int.Parse(Console.ReadLine());
            if(num%2==0) par++;
            else impar++;

            if(num>0) positivo++;
            else if(num<0) negativo++;
        }
        Console.WriteLine($"{par} valor(es) par(es)");
        Console.WriteLine($"{impar} valor(es) impar(es)");
        Console.WriteLine($"{positivo} valor(es) positivo(s)");
        Console.WriteLine($"{negativo} valor(es) negativo(s)");
    }

}